src/App.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { SimpleCard } from './SimpleCard';
+import { makeStyles, createStyles, Theme, useTheme } from '@material-ui/core/styles';
+import { Typography, Button, Fab } from '@material-ui/core';
+import * as Colors from '@pxblue/colors';
+import { Feedback } from '@material-ui/icons';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+import ellipseBackgroundImage from './assets/bg.png';
+import angularImage from './assets/angular-ionic-logo.svg';
+import reactImage from './assets/react-logo.svg';
+import reactNativeImage from './assets/react-native-logo.png';
+import titleImage from './assets/title.svg';
+import titleImageSM from './assets/titleSM.svg';
+
+const cardShift = -100;
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            display: 'flex',
+            flexDirection: 'column',
+            height: '100vh',
+        },
+        content: {
+            flex: '1',
+        },
+        ellipseBackground: {
+            background: `url(${ellipseBackgroundImage}) ${theme.palette.primary.main}`,
+            backgroundSize: 'cover',
+            clipPath: 'ellipse(70% 200px at 50% 50px)',
+            width: '100%',
+            height: 250,
+        },
+        SimpleCardContainer: {
+            margin: '0 auto',
+            justifyContent: 'center',
+            position: 'relative',
+            top: cardShift,
+            display: 'flex',
+        },
+        title: {
+            backgroundImage: `url("${titleImage}")`,
+            backgroundSize: 'contain',
+            backgroundRepeat: 'no-repeat',
+            backgroundPosition: 'center',
+            position: 'absolute',
+            top: theme.spacing(4),
+            left: '50%',
+            transform: 'translateX(-50%)',
+            maxWidth: 500,
+            width: '90%',
```

Comment:
Why 90%? Percentages have funny behavior when resizing. Can we use fixed margins to stay in line with the material keylines?